src/Platform/Microsoft.Testing.Platform/CommandLine/PlatformCommandLineProvider.cs

```diff
@@ -59,6 +60,7 @@ internal sealed class PlatformCommandLineProvider : ICommandLineOptionsProvider
         new(DiscoverTestsOptionKey, PlatformResources.PlatformCommandLineDiscoverTestsOptionDescription, ArgumentArity.Zero, false, isBuiltIn: true),
         new(IgnoreExitCodeOptionKey, PlatformResources.PlatformCommandLineIgnoreExitCodeOptionDescription, ArgumentArity.ExactlyOne, false, isBuiltIn: true),
         new(ExitOnProcessExitOptionKey, PlatformResources.PlatformCommandLineExitOnProcessExitOptionDescription, ArgumentArity.ExactlyOne, false, isBuiltIn: true),
+        new(ConfigOptionKey, PlatformResources.PlatformCommandLineConfigOptionDescription, ArgumentArity.ExactlyOne, false, isBuiltIn: true),
```

Comment:
I would validate the existance of the file inside the `ValidateOptionArgumentsAsync` so in case of we will have a meaninful message instead of an exception thrown below.